stories/documentation/navigation/menu/menu-basic.stories.ts

```diff
@@ -19,36 +19,35 @@ export default {
 				type: 'boolean',
 			}
 		},
-		palette: {
-			options: ['', 'palette-primary', 'palette-secondary', 'palette-grey', 'palette-success', 'palette-warning', 'palette-error'],
+		small: {
 			control: {
-				type: 'radio',
+				type: 'boolean',
 			}
 		},
 	},
 } as Meta;
 
 function getTemplate(args: MenuBasicStory): string {
-	const classes = [args.palette].filter(Boolean).join(' ');
 	const noBorder = args.noBorder ? `mod-noBorder` : '';
 	const header = args.header ? `mod-header` : '';
+	const small = args.small ? `mod-small` : '';
 	return `
-	<nav class="menu ${classes} ${noBorder} ${header}">
+	<nav class="menu ${small} ${noBorder} ${header}">
 		<ul class="menu-list">
 			<li class="menu-list-item">
-				<a href="#" class="menu-list-item-action" aria-current="page">
+				<button class="menu-list-item-action" aria-current="page">
```

Comment:
Why do you switch to buttons?

Reply:
Because I forget to revert my test.
![giphy](https://user-images.githubusercontent.com/25581936/164256828-6dde5663-aed4-48b7-99f0-9d7db967b8fd.gif)
Fixed.